dist/app/modules/MainScreen/MainScreen.js

```diff
@@ -0,0 +1,18 @@
+import React, { PureComponent } from 'react';
+import { MainWrapper, MainTitle } from './styles';
+import { Button } from 'react-native';
+import Modules from '../index';
+class MainScreen extends PureComponent {
+    constructor() {
+        super(...arguments);
+        this.handleToSecondScreen = () => {
+            this.props.navigation.navigate(Modules.ProfileScreenModule.name);
+        };
+    }
+    render() {
+        return (React.createElement(MainWrapper, null,
+            React.createElement(MainTitle, null, "Espeo Create React Native App"),
```

Comment:
all texts shoudl be in react intl. in separate language files. React intl will be done as a next thing ? BtW soonner the better, less places to be refactored later on.

Reply:
I see its dist catalog :) Please add it to gitignore and remove :)